plugins/tiddlywiki/multiwikiserver/modules/routes/handlers/change-user-password.js

```diff
@@ -22,28 +22,44 @@ exports.bodyFormat = "www-form-urlencoded";
 exports.csrfDisable = true;
 
 exports.handler = function (request, response, state) {
+  // Clean up any existing error/success messages
+  $tw.mws.store.adminWiki.deleteTiddler("$:/temp/mws/change-password/error");
+  $tw.mws.store.adminWiki.deleteTiddler("$:/temp/mws/change-password/success");
+  $tw.mws.store.adminWiki.deleteTiddler("$:/temp/mws/login/error");
+
   if(!state.authenticatedUser) {
-    response.writeHead(401, "Unauthorized", { "Content-Type": "text/plain" });
-    response.end("Unauthorized");
+    $tw.mws.store.adminWiki.addTiddler(new $tw.Tiddler({
```

Comment:
What would happen if two users were attempting to change password at the same time? Wouldn't `$:/temp/mws/login/error` get overwritten?